owasp-top10-2017-apps/a6/misconfig-wordpress/README.md

```diff
@@ -1,42 +1,227 @@
 # Vulnerable Wordpress Misconfig
-> This is a simple Wordpress web application that contains an example of a Security Misconfiguration vulnerability.
 
 <p align="center">
     <img src="images/banner.png"/>
 </p>
 
-## What is Security Misconfiguration?
+This is a simple Wordpress web application that contains an example of a Security Misconfiguration vulnerability and it's main goal is to describe how a malicious user could exploit multiple Security Misconfiguration vulnerabilities intentionally installed on SecWeb.
+
+## Index
 
-Definition from [OWASP](https://www.owasp.org/images/7/72/OWASP_Top_10-2017_%28en%29.pdf.pdf):
+- [Definition](#what-is-security-misconfiguration)
+- [Setup](#setup)
+- [Attack narrative](#attack-narrative)
+- [Objectives](#secure-this-app-🔧)
```

Comment:
I just noticed the `Objectives` link is broken because of the 🔧 emoji. I guess you'll need to delete the poor emoji, unfortunately 😥

Reply:
Nice catch! Thanks @Takehime !